test/spec/OktaAuth/api.ts

```diff
@@ -12,6 +12,23 @@
 
 
 /* eslint-disable no-new */
+
+const mocked = {
+  findKeyPair: jest.fn(),
+  // generateDPoPProof: jest.fn(),
+  clearDPoPKeyPair: jest.fn(),
+  clearAllDPoPKeyPairs: jest.fn(),
+  clearDPoPKeyPairAfterRevoke: jest.fn(),
+};
+jest.mock('../../../lib/oidc/dpop', () => {
+  const actual = jest.requireActual('../../../lib/oidc/dpop');
+  return {
+    __esModule: true,
+    ...actual,
+    ...mocked
+  };
+});
```

Comment:
interesting. any notes?

Reply:
Any note in particular you're looking for? I want to DPoP methods to be tested, but db wrappers (like findKeyPair) can be mocked

Reply:
guess not. unusual syntax so i was just wondering if there's anything to watch out for